File: 1880-check-if-word-equals-summation-of-two-words/1880-check-if-word-equals-summation-of-two-words.kt
class Solution {
    fun sum(word: String): Int {
        var res = 0
        for (i in 0..(word.length-1)) {
            res *= 10
            res += (word[i] - 'a')
        }
        return res
    }
    
    fun isSumEqual(firstWord: String, secondWord: String, targetWord: String): Boolean {
        return sum(firstWord) + sum(secondWord) === sum(targetWord)
    }
}

File: 1880-check-if-word-equals-summation-of-two-words/1880-check-if-word-equals-summation-of-two-words.py
class Solution:
    def isSumEqual(self, firstWord: str, secondWord: str, targetWord: str) -> bool:
        def helper(word: str) -> int:
            abc = 'abcdefghijklmnopqrstuvwxyz'
            res = ''
            for c in word:
                for i in range(0, len(abc)-1):
                    if c == abc[i]:
                        res += str(i)
                        break
            while res and res[0] == "0":
                res = res[1::]
            return int(res) if res else 0
        
        return helper(firstWord) + helper(secondWord) == helper(targetWord)